plugins/woocommerce-admin/client/tasks/fills/Marketing/index.tsx

```diff
@@ -55,39 +56,43 @@ export const getMarketingExtensionLists = (
 ): [ PluginProps[], PluginListProps[] ] => {
 	const installed: PluginProps[] = [];
 	const lists: PluginListProps[] = [];
-	const freeExtensionsRandomized: ExtensionList[] = freeExtensions.sort(
-		() => Math.random() - 0.5
-	); // Randomize the order sections appear.
-
-	freeExtensionsRandomized.forEach( ( list ) => {
-		if ( ! ALLOWED_PLUGIN_LISTS.includes( list.key ) ) {
-			return;
-		}
-
-		const listPlugins: PluginProps[] = [];
-		list.plugins.forEach( ( extension ) => {
-			const plugin = transformExtensionToPlugin(
-				extension,
-				activePlugins,
-				installedPlugins
+
+	freeExtensions
+		.sort( ( a: ExtensionList, b: ExtensionList ) => {
+			return (
+				PLUGIN_LIST_DISPLAY_ORDER.indexOf( a.key ) -
+				PLUGIN_LIST_DISPLAY_ORDER.indexOf( b.key )
 			);
-			if ( plugin.isInstalled ) {
-				installed.push( plugin );
+		} )
+		.forEach( ( list ) => {
```

Comment:
I only removed the random freeExtensionsRandomized and added `.sort()` here. The changes below (line 67-95) are automatically formatted by linter.